README.md

```diff
@@ -1,6 +1,11 @@
 # mozlogrus [![GoDoc](https://godoc.org/go.mozilla.org/mozlogrus?status.svg)](https://godoc.org/go.mozilla.org/mozlogrus)
 A logging library which conforms to [Mozilla's logging standard](https://wiki.mozilla.org/Firefox/Services/Logging) for [logrus](https://github.com/Sirupsen/logrus).
 
+## Installation
+
+* ver 1: `go get gopkg.in/mozilla-services/go-mozlog.v1`
+* ver 2: `go get gopkg.in/mozilla-services/go-mozlog.v2`
```

Comment:
Why not use `go.mozilla.org` like the rest of our go stuff?

Reply:
Because go.mozilla.org doesn't support pulling versions from branches/tags. gopkg.in does.

Reply:
What mechanism is required to make that happen?
I'd rather not have a core library depend on an external service we don't have a contractual relationship with, if possible.

Reply:
I think gopkg.in works as a proxy to the git checkout stuff.  I haven't dug through the code but it's like magic. :) 

There's several options here: 

1. use gopkg.in
2. run our own gopkg.in (the code is opensource)
3. move V2 to a new repo, github.com/mozilla-services/go-mozlogrus-v2

The third option is better for `go get` which expects `HEAD` to always be the latest, api compatible stable version. 

@oremj what do you think?

Reply:
Yeah, gopkg.in understands enuogh of the git URLs to redirect them appropriately. `go.mozilla.org` is just a static s3 bucket with html files, so I don't think we can do the same redirects there.

I like option 3. It's cleaner in the long run.

Reply:
I'm happy to create a new go-mozlog-v2 repo with this PR. If oremj is good with it.

Reply:
We probably don't need a v2 repo, the only possible breaking change here is the addition of the Type field and it will only break people if they have declared the Formatter manually.

Reply:
This is likely the complete list of projects using mozlogrus: https://github.com/search?q=mozlogrus&type=Code&utf8=%E2%9C%93

Use `Enable`:
* reaper
* go-toybox
* iam
* autograph
* tigerblood
* stubattribution

Does not use `Enable`:
* taskcluster/webhooktunnel

Reply:
We could submit a PR to webhooktunnel or add app.log as a default. For example, in `Format`:
```
if m.Type == "" {
    m.Type = "app.log"
}
```

Reply:
I think 

```
if m.Type == "" {
    m.Type = "app.log"
}
```

would be the most pragmatic approach. This way we can make sure we are totally V1 backwards compatible and don't have to worry about downstream at all.